src/fetchtastic/menu_repo.py

```
@@ -57,7 +70,7 @@ def _is_action_option(self, option: Option) -> bool:
             return False
         return option.value.get("type") in {"back", "quit"}
 
-    def run_loop(self, screen: "curses._CursesWindow", position: "Position") -> Any:
+    def run_loop(self, screen: CursesScreen, position: "Position") -> Any:
```

Ruff (0.14.10)

Unused method argument: `position`

(ARG002)